src/pages/articles/bootstrap/bootstrap-get-started/index.md

```diff
@@ -2,14 +2,28 @@
 title: Bootstrap Get Started
 ---
 ## Bootstrap Get Started
+When using Bootstrap for the first time it can appear to be a confusing process, but using Bootstrap is actually quite simple. Using Bootstrap makes it easy to design a fully responsive website and is a framework worth learning.
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-get-started/index.md).
+#### What is a responsive website:
+A responsive website is a website that resizes and rearranges that items on the page depending on the size of your browser. With a responsive website if you resize your browser you can see the changes occur in real time. Bootstrap makes your website responsive for you.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+#### How do I add Bootstrap to my page:
+Adding bootstrap to your page is quite simple, you simply add the following to the `<head>` tags in your code.
```

Comment:
If someone is struggling getting Bootstrap to work, this line will not help them. Try to not use simply